app/views/users/_customization.html.erb

```diff
@@ -20,7 +21,8 @@
     <div class="crayons-field">
       <%= f.label :config_font, "Base Reading Font", class: "crayons-field__label" %>
       <div class="theme-selector-field grid gap-4 grid-cols-2 l:grid-cols-3">
-        <% User::FONTS.each do |font| %>
+        <!-- [@msarit] we've included default here, double chcek the behaviour -->
```

Comment:
The behavior is thus:
1. User selects `default` and saves.
2. The option selected is `sans_serif` which is the default font stored in `Settings::UserExperience.default_font`.